docs/source/write_first_app.rst

```diff
@@ -101,889 +97,485 @@ bringing up a new one.
-      const ccp = buildCCP();
-
-      // build an instance of the fabric ca services client based on
-      // the information in the network configuration
-      const caClient = buildCAClient(FabricCAServices, ccp);
-
-      // setup the wallet to hold the credentials of the application user
-      const wallet = await buildWallet(Wallets, walletPath);
-
-      // in a real application this would be done on an administrative flow, and only once
-      await enrollAdmin(caClient, wallet);
-
-This command stores the CA administrator's credentials in the ``wallet`` directory.
-You can find administrator's certificate and private key in the ``wallet/admin.id``
-file.
+``asset-transfer-basic/application-gateway-typescript`` directory, run the following command:
 
-.. note:: If you decide to start over by taking down the network and bringing it back up again, you will
-          have to delete the ``wallet`` folder and its identities prior to re-running the javascript application
-          or you will get an error. This happens because the Certificate Authority and its database are taken down
-          when the test-network is taken down but the original wallet still remains in the application-javascript directory
-          so it must be deleted.  When you re-run the sample javascript application, a new wallet and credentials will
-          be generated.
+.. code-block:: bash
 
+  npm start
 
-If you scroll back up to the beginning of the output in your terminal, it should be similar to below:
 
-.. code:: bash
-
-  Wallet path: /Users/<your_username>/fabric-samples/asset-transfer-basic/application-javascript/wallet
-  Successfully enrolled admin user and imported it into the wallet
-
-Because the admin registration step is bootstrapped when the Certificate Authority
-is started, we only need to enroll the admin.
-
-.. note:: Since the Fabric CA interactions are common across the samples, enrollAdmin() and the other CA
-          related functions are included in the ``fabric-samples/test-application/javascript/CAUtil.js``
-          common utility.
-
-As for the app user, we need the application to register and enroll the user in the next step.
-
-
-Second, the application registers and enrolls an application user
------------------------------------------------------------------
+First, the application creates a gRPC connection to the Gateway peer
+--------------------------------------------------------------------
 
-Now that we have the administrator's credentials in a wallet, the application uses the ``admin``
-user to register and enroll an app user which will be used
-to interact with the blockchain network. The section of the application code is shown below.
+The client application establishes a `gRPC <https://grpc.io/>`__ connection to the Gateway peer that it will use to
+transact with the network. To do this it only requires the endpoint address of the Gateway peer and, if the Gateway is
```

Comment:
do you want to say "with the channeL" here instead of "with the network"? technically it's the channel right, which may not be the entire network of channels.

Reply:
and also "target peer and, if the gateway is ..."

Reply:
No, it doesn't restrict the client to interacting with a single channel.